packages/contracts-bedrock/src/Safe/VetoerSet/AddOwnerModule.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Safe } from "safe-contracts/Safe.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+
+import { ISemver } from "src/universal/ISemver.sol";
+
+import { OwnerGuard } from "./OwnerGuard.sol";
+
+/// @title AddOwnerModule
+/// @notice This module allows any specifically designated address to add owners to the
+///         Safe Account. Specifically, the Optimism Foundation may add new owners.
+contract AddOwnerModule is ISemver {
+    /// @notice Semantic version.
+    /// @custom:semver 1.0.0
+    string public constant version = "1.0.0";
+
+    /// @notice The Safe contract instance
+    Safe internal immutable safe;
+
+    /// @notice The OwnerGuard instance
+    OwnerGuard internal immutable ownerGuard;
+
+    /// @notice The OP Foundation multisig address
+    address internal immutable opFoundation;
```

Comment:
Perhaps we could decouple this from "OP Foundation" specific terminology and instead make the name based more on the functionality? The spec doc could include information on how the module is configured in practice